script.js

```diff
@@ -1,4 +1,131 @@
+var currentGameMode = "Waiting for next player";
+var playerScores = [];
+var round = 0;
+var maxNum = 0;
+var noOfPlayers = 3;
+var diceRoll1 = 0;
+var diceRoll2 = 0;
+var scoreBoard = [];
+
+var diceRoll = function () {
+  var randomNum = Math.random() * 6;
+  var randomInt = Math.floor(randomNum) + 1;
+
+  return randomInt;
+};
+
+var concatNum = function (num1, num2, input) {
+  var num = 0;
+  if (input == 1) {
+    num = parseInt(String(num1) + String(num2));
+  }
+
+  if (input == 2) {
+    num = parseInt(String(num2) + String(num1));
+  }
+
+  return num;
+};
+
+var findSortedArray = function (arr) {
+  var originalIndexVal = [];
+
+  for (var i = 0; i < arr.length; i++) {
+    originalIndexVal.push({ player: i, score: arr[i] });
+  }
+
+  var sortedArray = originalIndexVal.sort(function (a, b) {
+    return b.score - a.score;
+  });
+
+  return sortedArray;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "Press 'Submit' to start rolling the dice!";
+
+  while (round < noOfPlayers) {
+    console.log(round);
+    if (currentGameMode == "Waiting for next player") {
+      if (input != "") {
```

Comment:
I think the way you used if and else if is pretty neat and clear. However, I encourage you to use early return for blackjack to further improve the ease of reading of your code!